pkg/container/bubblewrap_runner.go

```diff
@@ -85,9 +85,13 @@ func (bw *bubblewrap) cmd(ctx context.Context, cfg *Config, debug bool, envOverr
 		"--chdir", runnerWorkdir,
```

Comment:
here we can add `--cap-add` to address the problems
Ideally we should use the minimum used by docker, so we should add here:

```go
"--cap-add", "CAP_CHOWN",
"--cap-add", "CAP_DAC_OVERRIDE",
"--cap-add", "CAP_FSETID",
"--cap-add", "CAP_FOWNER",
"--cap-add", "CAP_MKNOD",
"--cap-add", "CAP_NET_RAW",
"--cap-add", "CAP_SETGID",
"--cap-add", "CAP_SETUID",
"--cap-add", "CAP_SETFCAP",
"--cap-add", "CAP_SETPCAP",
"--cap-add", "CAP_NET_BIND_SERVICE",
"--cap-add", "CAP_SYS_CHROOT",
"--cap-add", "CAP_KILL",
"--cap-add", "CAP_AUDIT_WRITE",
```

Reply:
Reference https://github.com/moby/moby/blob/master/oci/caps/defaults.go#L6-L19